spark/dl/src/main/scala/com/intel/analytics/bigdl/models/vgg/VggForCifar10.scala

```diff
@@ -341,6 +344,6 @@ object Vgg_19 {
     val linear3 = Linear(4096, classNum).inputs(drop2)
     val output = LogSoftMax().inputs(linear3)
 
-    Graph(conv1, output)
+    if(isDynamic) Graph.dynamic(conv1, output) else Graph(conv1, output)
```

Comment:
Why do we need isDynamic for the models/examples? It should always be static graph, no?

Reply:
This is for test... We have a set of test for these graph models, so I want to test both dynamic and static